spring-boot-project/spring-boot-docs/src/docs/antora/modules/how-to/pages/properties-and-configuration.adoc

```diff
@@ -209,7 +209,7 @@ spring.datasource.url=jdbc:mysql://localhost/test
 server.port=9000
 ----
 
-See xref:reference:features/external-config.adoc#features.external-config.yaml[] in the "`Spring Boot Features'" section for more information about YAML.
+See xref:reference:features/external-config.adoc#features.external-config.yaml[] in the "`Spring Boot Features`" section for more information about YAML.
```

Comment:
<img width="906" alt="Screenshot 2024-11-23 at 7 31 23 PM" src="https://github.com/user-attachments/assets/dec68643-443b-4f98-984c-6295ece6b122">

curious .. will this change update the font of text, which is not the case I guess?